cli/dts/lib.deno.unstable.d.ts

```diff
@@ -835,6 +835,54 @@ declare namespace Deno {
     mtime: number | Date,
   ): Promise<void>;
 
+  /** The type of the resource record.
+   * Only the listed types are supported currently. */
+  export type RecordType =
+    | "A"
+    | "AAAA"
+    | "ANAME"
+    | "CNAME"
+    | "MX"
+    | "PTR"
+    | "SRV"
+    | "TXT";
+
+  export interface ResolveDnsOptions {
+    /** The name server to be used for lookups. 
+    * If not specified, defaults to the system configuration e.g. `/etc/resolve.conf` on Unix. */
```

Comment:
```suggestion
    * If not specified, defaults to the system configuration e.g. `/etc/resolv.conf` on Linux. */
```

macOS doesn't actually use `/etc/resolv.conf`.

Reply:
resolv.conf is a user land configuration. Are you saying that trust dns doesn’t look at resolve.conf in OSX?

Reply:
Oh, I wrote `resolve.conf` mistakenly. Thanks for pointing out :)

https://github.com/bluejekyll/trust-dns/blob/main/crates/resolver/src/system_conf/unix.rs#L29-L31
According to the implementation, it seems like `trust-dns` looks at `/etc/resolv.conf` in macOS too
as this `read_system_conf` function is under `#[cfg(unix)]`.
(implementation for windows is [here](https://github.com/bluejekyll/trust-dns/blob/main/crates/resolver/src/system_conf/windows.rs#L51))